sdk/sdk/src/main/java/org/apache/carbondata/sdk/file/JsonCarbonWriter.java

```diff
@@ -91,4 +106,44 @@ public void close() throws IOException {
       throw new IOException(e);
     }
   }
+
+  private void loadSingleFile(CarbonFile file) throws IOException {
+    Reader reader = null;
+    try {
+      reader = SDKUtil.buildJsonReader(file, configuration);
+      JSONParser jsonParser = new JSONParser();
+      Object jsonRecord = jsonParser.parse(reader);
+      if (jsonRecord instanceof JSONArray) {
+        JSONArray jsonArray = (JSONArray) jsonRecord;
+        for (Object record : jsonArray) {
+          this.write(record.toString());
+        }
+      } else {
+        this.write(jsonRecord.toString());
+      }
+    } catch (Exception e) {
```

Comment:
Use specific exceptions instead of generic exception

Reply:
Done.